### ISSUE TITLE
Ignored Files seemingly not ignored when configured through UI config

### ISSUE THREAD

**Issue by Bodge-IT** [dummylabs/thewatchman#27]
The ignored files doesn't seem to work. I've tried leading slash, no leading slash, leading '/config' no leading '/config', wrap in "", no wrap in "", path/to/file, no/path/to/file, wildcards (in "" of course, I RTFM'd). I could not get either of my files to be excluded. Fell back to individual entities excluded for the win.

v0.5.0
Configured through UI, not in yaml.
Great tool, thanks.

**Comment by chertvl** [comment]
This variant works for me:
/config/custom_components/*, /config/www/*, /config/esphome/*, /config/nodered/*, */customize.yaml

**Comment by dummylabs** [comment]
Hey @Bodge-IT, did you manage to configure ignored files properly? If not, can you set debug logging for the integration and provide following information from the log:
1. Home assistant path (your config root, e.g. /config)
`2022-04-27 19:51:31 DEBUG (MainThread) [custom_components.watchman] Home assistant path: /config/`
2. Your actual settings like this: 
`2022-04-27 19:51:31 DEBUG (MainThread) [custom_components.watchman] {'service': '', 'service_data': '{}', 'included_folders': [], 'report_header': '-== Watchman Report ==-', 'report_path': '', 'ignored_items': [], 'ignored_states': [], 'chunk_size': 3500, 'ignored_files': [], 'check_lovelace': False, 'columns_width': [30, 7, 60], 'startup_delay': 0, 'friendly_names': False}`
4. The name of a file which should be ignored, but was parsed instead (from the debug log), e.g:
`2022-04-27 19:51:32 DEBUG (MainThread) [custom_components.watchman.utils] /config/custom_components/watchman/fake_sensors.yaml parsed`

The debug log level for watchman can be set like this:
```yaml
logger:
  default: info
  logs:
    custom_components.watchman: debug
```

**Comment by Bodge-IT** [comment]
I didn't, but as I had the fallback of individual elements, I took that route.  Weekends coming up, I'll give it another go and pass back the info requested.

**Comment by noogugae** [comment]
It's same for me.
I have two HA, one is on RPI4b and the other is on VMM of DS920+. 
in case of DS920+, it works well. but In case of RPI4b, it doesn't work.

I added "/config/lovelace/button_card_templates/button_card_templates.yaml" but it still reports the error.
Also there is one strange ones.
Firstly it reports "lovelace/button_card_templates/button_card_templates. yaml" have  missing entities.
but after I added that path. it start to report "config/lovelace/button_card_templates/button_card_templates.yaml" have missing entities.

this is my 'watchman_report.txt'
It said there is 1 ignored file but watchman_report still report missing case of the ignored file.

| sensor.xiaomi_multisensor_salo        | missing | config/lovelace/button_card_templates/button_card_templates. |
| n_humidite                            |         | yaml:1269                                                    |
+---------------------------------------+---------+--------------------------------------------------------------+

-== Report created on 30 Apr 2022 23:05:44
-== Parsed 12012 files in 60.36s., ignored 1 files 
-== Generated in: 0.06s. Validated in: 0.00s.

**Comment by Bodge-IT** [comment]
OK, here is the entry in the report:
`| climate.hvac                                       | missing  | custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/popup_templates/popup_buttons/popup_button_airconditionner.yaml`
here is my entry in config:
"/config/custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/popup_templates/popup_buttons/popup_button_airconditionner.yaml"

And I have removed the entity from the ignored entity field. Nothing showed up in logs.

**Comment by Bodge-IT** [comment]
Aha, re-read your instructions and looked for the parsing lines on the files above and found this:
2022-05-02 11:28:34 DEBUG (MainThread) [custom_components.watchman.utils] /config/custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_vacuum.yaml ignored
And this:
2022-05-02 11:28:34 DEBUG (MainThread) [custom_components.watchman.utils] /config/custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/popup_templates/popup_buttons/popup_button_airconditionner.yaml ignored

| vacuum.stop                                        | missing  | custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_vacuum.yaml:162,170  |
| vacuum.start                                       | missing  | custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_vacuum.yaml:172      |
| vacuum.return_to_base                              | missing  | custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_vacuum.yaml:185      |
| vacuum.locate                                      | missing  | custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_vacuum.yaml:197      |
| climate.hvac                                       | missing  | custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/popup_templates/popup_buttons/popup_button_air |
|                                                    |          | conditionner.yaml:22 

Still getting this in report:
-== Report created on 02 May 2022 11:00:52
-== Parsed 317 files in 0.72s., ignored 0 files 
-== Generated in: 0.03s. Validated in: 0.01s.

**Comment by dummylabs** [comment]
Thanks for the feedback, good to see it works for you now.
> Maybe your scanner not always following symlinks?

Watchman uses Python standard library to open files, it relies on underlying operating system services. Though, it may crash when a dead link encountered, then the parsing will fail. This will be fixed in #37. 
>Also, using "" not recommended.

You're right, I'll check the docs. Quotes were mandatory for yaml configuration, when your search pattern has leading `*`, otherwise Home Assistant's parser complained about a configuration file error. They're unnecessary for UI configuration.
